apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDataAdapter.cs

```diff
@@ -155,6 +161,11 @@ IDbCommand IDbDataAdapter.UpdateCommand
         }
 
         /// <summary>
+        /// The set of primary keys defined for <see cref="AutoGeneratedCommandTable"/>.
+        /// </summary>
+        public HashSet<string> AutoGeneratedCommandPrimaryKeys { get; } = new HashSet<string>();
```

Comment:
yes for the case where the developer uses the default ctor (which is the designer scenario).
Not sure if we'll be able to get the toolbox working near term, but I don't want to cut off the use case.